File: DayOne/java/TaskTen.java
import java.util.Scanner;
public class TaskTen{
	public static void main(String[] args){
		Scanner input = new Scanner(System.in);
		int adding = 0;
		int counter = 1;
		for(int count = 0; count<10;count++){
			System.out.println("Enter score: ");
			int score = input.nextInt();
			if(score>0 && score<100){
				adding += score;
				counter++	
			}
		}
		int average = adding/counter;
		System.out.print(average);
	}
}

File: DayOne/python/task_ten.py
count = 0;
TEN = 10
add = 0;
counter = 0;
while count<TEN:
	score = int(input('Enter score: ')
	if(score>0 and score<100):
		add+=score
		counter+=1
	count+=1
average = add/counter
print(add)
print(average)